File: Modul-3/Python/PRAK305-2310817320007-ZahraNabila.py
Detik = int(input("Masukkan Jumlah Detik = "))

Hari = Detik // ( 24 * 3600 )
Jam = ( Detik % ( 24 * 3600 )) // 3600
Menit = ( Detik % 3600 ) // 60
sisaDetik = Detik % 60

if Detik >= 86400:
    print(f'Konfersi Waktu        = {Hari} hari {Jam:02d}:{Menit:02d}:{sisaDetik:02d}')

else:
    print(f'Konfersi Waktu        = {Jam:02d}:{Menit:02d}:{sisaDetik:02d}')

File: Modul-3/C/PRAK305-2310817320007-ZahraNabila.c
#include <stdio.h>

int main() 
{
    int Hari, Jam, Menit, Detik, sisaDetik;

    printf("Masukkan Jumlah Detik = ");
    scanf("%d", &Detik);

    Hari = Detik / ( 24 * 3600 );
    Jam = ( Detik % ( 24 * 3600 )) / 3600;
    Menit = ( Detik % 3600 ) / 60;
    sisaDetik = Detik % 60;

        if (Detik >= 86400)
        {
            printf("Konfersi Waktu        = %d hari %02d:%02d:%02d\n", Hari, Jam, Menit, sisaDetik);
        }
        else
        {
            printf("Konfersi Waktu        = %02d:%02d:%02d\n", Jam, Menit, sisaDetik);
        }
    return 0;
}